src/EditorFeatures/Core.Cocoa/Preview/PreviewPaneService.cs

```diff
@@ -101,7 +101,6 @@ private NSImage GetSeverityIconForDiagnostic(DiagnosticData diagnostic)
             if (data.Properties.TryGetValue("OptionName", out _))
             {
                 data.Properties.TryGetValue("OptionLanguage", out _);
-                throw new NotImplementedException();
```

Comment:
❔ What does EditorFeatures.Wpf do for this?

Reply:
http://sourceroslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/PreviewPane/PreviewPaneService.cs,102

Reply:
From the looks of it, opens a Global Options page with said option, which we don't expose in VSMac.

I found this when previewing `Fix formatting` code fix.